tests/test_matrix.py

```diff
@@ -173,3 +189,34 @@ def test_multiplyMatrix():
     # assert matPro.__str__() == ""
     """
     pass
+
+
+#################
+# matrix.structure #
+#################
+
+
+def test_transposeMat():
+
+    mat = getTokens("[1, 2; \
+                    3,  4]")
+    matTranspose = Matrix.transposeMat(mat)
+    assert matTranspose.__str__() == "[{1.0},{3.0};{2.0},{4.0}]"
+
+    mat = getTokens("[5,8,2;\
+                      12,30,9;\
+                      4,17,7]")
+    matTranspose = Matrix.transposeMat(mat)
+    assert matTranspose.__str__() == "[{5.0},{12.0},{4.0};{8.0},{30.0},{17.0};{2.0},{9.0},{7.0}]"
+
+
+def test_isSquare():
+
+    mat = getTokens("[1, 2; \
+                      x, z]")
+    assert Matrix.isSquare(mat)
```

Comment:
Use `mat.isSquare()` instead of `Matrix.isSquare(mat)`. Same in line 222 as well.
```suggestion
    assert mat.isSquare()
```